guides/STAKING-VALIDATOR.md

```diff
@@ -0,0 +1,308 @@
+const networkName =
+  solanaConfig.get<svmkit.solana.NetworkName>("network") ??
+  svmkit.solana.NetworkName.Devnet;
+const networkInfo = svmkit.networkinfo.getNetworkInfoOutput({ networkName });
+
+// Create some keys for this validator to use.
+const validatorKey = new svmkit.KeyPair("validator-key");
+const treasuryKey = new svmkit.KeyPair("treasury-key");
+const voteAccountKey = new svmkit.KeyPair("vote-account-key");
+const stakeAccountKey = new svmkit.KeyPair("stake-account-key");
+
+// Point pulumi-svmkit at the AWS EC2 instance's SSH connection.
+const connection = {
+  host: instance.publicDns,
+  user: "admin",
+  privateKey: sshKey.privateKeyOpenssh,
+};
+
+// Tuner setup
+const tunerVariant =
+    tunerConfig.get<svmkit.tuner.TunerVariant>("variant") ??
+    svmkit.tuner.TunerVariant.Generic;
+
+// Retrieve the default tuner parameters for that variant
+const genericTunerParamsOutput = svmkit.tuner.getDefaultTunerParamsOutput({
+  variant: tunerVariant,
+});
+
+// "Apply" those params so we can pass them to the Tuner constructor
+const tunerParams = genericTunerParamsOutput.apply((p) => ({
+  cpuGovernor: p.cpuGovernor,
+  kernel: p.kernel,
+  net: p.net,
+  vm: p.vm,
+  fs: p.fs,
+}));
+
+// Create the Tuner resource on the EC2 instance
+const tuner = new svmkit.tuner.Tuner(
+  "tuner",
+  {
+    connection,
+    params: tunerParams,
+  },
+  {
+    dependsOn: [instance],
+  }
+);
+
+// Create vote account
+const voteAccount = new svmkit.account.VoteAccount("validator-vote-account", {
+  connection: connection,
+  keyPairs: {
+    identity: validatorKey.json,
+    voteAccount: voteAccountKey.json,
+    authWithdrawer: treasuryKey.json,// Treasury key as the withdraw authority
+  },
+}, { dependsOn: [instance] });
+
+// Create and delegate stake account
+const stakeAccount = new svmkit.account.StakeAccount("validator-stake-account", {
+    connection: connection,
+    transactionOptions: {
+      keyPair: treasuryKey.json,
+    },
+    keyPairs: {
+      stakeAccount: stakeAccountKey.json,
+      voteAccount: voteAccountKey.json,
+    },
+    amount: 0.2, // Stake amount in SOL
+}, { dependsOn: [voteAccount] });
+
+// Instantiate a new Agave instance on the machine.
+new svmkit.validator.Agave(
+  "validator",
+  {
+    connection,
+    version: "2.1.13-1",
+    environment: {
+      rpcURL: networkInfo.rpcURL[0],
+    },
+    keyPairs: {
+      identity: validatorKey.json,
+      voteAccount: voteAccountKey.json,
+    },
+    flags: {
+      useSnapshotArchivesAtStartup: "when-newest",
+      fullRpcAPI: false,
+      rpcPort: 8899,
+      privateRPC: true,
+      onlyKnownRPC: true,
+      dynamicPortRange: "8002-8020",
+      gossipPort: 8001,
+      rpcBindAddress: "0.0.0.0",
+      walRecoveryMode: "skip_any_corrupted_record",
+      limitLedgerSize: 50000000,
+      blockProductionMethod: "central-scheduler",
+      fullSnapshotIntervalSlots: 1000,
+      noWaitForVoteToStartLeader: true,
+      noVoting: true,
```

Comment:
```suggestion
      noVoting: false,
```

Reply:
Its time to vote :)

In terms of agave validator code can you replace this full example with another stope saying to switch from true to false on the validator client. Don't include the full code for the validator.